### ISSUE TITLE
[CoE Starter Kit - QUESTION] - [Child] Admin | Sync Logs - The subscription was disabled.

### ISSUE THREAD

**Issue by stefwanders** [microsoft/coe-starter-kit#4018]
### What is your question?

After installing and going through all the necessary steps of the Solution - Audit Logs, I am checking the FLOW runs.
**Note** I did not change any of the timing settings, left them to Default. Installation of CoE is on a Test Tenant (very small)

The FLOW: Admin | Sync Audit Logs ran once and Succeeded
Upon inspection of the FLOW: [Child] Admin | Sync Logs, I saw that the majority of the FLOWS succeeded, some failed with error message: The subscription was disabled.

Can anyone elaborate on this error message? 
![Sync logs FLOW1](https://user-images.githubusercontent.com/64784688/197201554-def4b379-eec7-438c-942a-d72aae0acae3.png)

![Sync logs FLOW2](https://user-images.githubusercontent.com/64784688/197201576-781ea2b3-50d8-4a42-b1cc-fec2c09a23c7.png)

I've checked the following things:
- Custom API connection under Dataverse. It's setup correctly, by testing I get a 400 error message: Subscription has already been setup.
- Before implementing I've checked unmanaged layers, there were none there. Now however, after the first runs, I see that there is an unmanaged layer: not sure if this is affecting the FLOW / error message.

![image](https://user-images.githubusercontent.com/64784688/197202006-ea8ef6c2-cca6-4fc6-8889-34f6245dea83.png)

Perhaps it is nothing, perhaps it is something? I'm worried my Audit logs will not give a good overview.

### What solution are you experiencing the issue with?

Audit Log

### What solution version are you using?

CeneterOfExcellenceAuditLogs)2.10_managed

### What app or flow are you having the issue with?

[Child] Admin | Sync Logs

**Comment by stefwanders** [comment]
Update: ran the Flows (by hand) one more time, activating the main flow and all child flows now ran successfully.

Trying again tomorrow.

**Comment by Jenefer-Monroe** [comment]
Are you running with an identity that is nor permanently assigned its roles / licenses via PIM? Sounds like it might be something like that. Note [we cannot support PIM](https://learn.microsoft.com/power-platform/guidance/coe/limitations#pim-privileged-identity-management)

**Comment by stefwanders** [comment]
Hello Jenefer; good questions; the account is a permanent global admin (it's our 'test' tenant). Has an A5 license for all necessary permissions in terms of Purview (audit). 

I think the next run is scheduled to run soon; I'll await this next run.

The only thing I can think of is that Audit log was not enabled at the time I created the app registration required. I turned it on with PS and tested the connection on the custom connector, which in the first few verifications was giving me a 400. Giving it some time, the test was successful on the Custom connector.

Perhaps it's all related by wanting to go too fast with things, yet searching the web on the term: Subscription was disabled, gave no related hits what so ever. 

I'll update the ticket after the next FLOW run.
